pykechain/client.py

```diff
@@ -2572,6 +2579,12 @@ def create_widget(
         :raises IllegalArgumentError: when an illegal argument is send.
         :raises APIError: when an API Error occurs.
         """
+        if widget_type == WidgetTypes.PROGRESS:
```

Comment:
This calls `validate_widget` which already has the same check on the widget_type

Reply:
okay - agreed